### DISCUSSION TITLE
Modifying schema for tables in dashboard SQL UI

### DISCUSSION THREAD

**Discussion by benzark** [supabase/supabase#31077] [Questions] [answered]
When I run queries like ALTER TABLE my_schema.my_table SET SCHEMA public; I get the message 'Success. No rows returned' but the table schema is not changed. Anyone else having this issue?

**Comment by GaryAustin1** [comment]
This also does not work from the direct data port with Datagrip.

**Reply by GaryAustin1** [reply]
Looks like it got broken and a fix is coming?
https://github.com/supabase/supautils/pull/100

**Reply by GaryAustin1** [reply, marked as answer]
I confirm upgrading instance to latest version fixes problem.

**Reply by rob-didio** [reply]
To elaborate on @GaryAustin1's answer. It seems like there was an issue changed in Postgres 15.6 related to catalog changes in a transaction. We had the same issue with a migration moving tables from a custom schema to the public schema. Upgrading the instance from 15.1 -> 15.8 (latest as of now) worked for us.

https://git.postgresql.org/gitweb/?p=postgresql.git;a=commitdiff;h=b793a416b
